qiskit_experiments/framework/analysis_result_table.py

```diff
@@ -156,11 +195,35 @@ def add_entry(
                     "experiment data. Please use another ID to avoid index collision."
                 )
 
-        return super().add_entry(
-            index=short_id,
-            result_id=result_id,
-            **kwargs,
-        )
+            return self._container.add_entry(
+                index=short_id,
+                result_id=result_id,
+                **kwargs,
+            )
+
+    def drop(
```

Comment:
I'd prefer `drop_entry` (To me, `drop` feels it implements all features supported in `pd.Datafram.drop`)

Reply:
Makes sense. Done in [6925655](https://github.com/Qiskit-Extensions/qiskit-experiments/pull/1252/commits/692565580bf290307095e8c8a24fefb50ab52722)